mysql-test/suite/component_masking_functions/t/flusher_thread_immediate_restart.test

```diff
@@ -0,0 +1,22 @@
+--source include/have_masking_functions_component.inc
+
+CALL mtr.add_suppression("Flusher thread terminated while waiting for session server");
+CALL mtr.add_suppression("Flusher thread terminated after creating internal connection");
+
+CREATE TABLE mysql.masking_dictionaries(
+    Dictionary VARCHAR(256) NOT NULL,
+    Term VARCHAR(256) NOT NULL,
+    UNIQUE INDEX dictionary_term_idx (Dictionary, Term)
+) ENGINE = InnoDB DEFAULT CHARSET=utf8mb4;
+GRANT SELECT, INSERT, UPDATE, DELETE ON mysql.masking_dictionaries TO 'mysql.session'@'localhost';
+INSERT INTO mysql.masking_dictionaries VALUES('dict1', 'term1');
+
+
+INSTALL COMPONENT 'file://component_masking_functions';
```

Comment:
OK. I guess this test will occasionally trigger scenario with immediate shutdown and component deinit failure I have mentioned... But I wonder how often it will happen... Perhaps we can make situation more reproducible by adding debug sleep at various stages of flusher thread initialization? What do you think?

Reply:
See previous response